beluga_demo_nav2/worlds/world_only.model

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0"?>
```

Comment:
@Alondruck meta: do we need all these things here when available upstream?

Reply:
I wanted to copy this file to avoid depending on updates from other repositories for this tutorial, since the only important thing here is the integration of `beluga_amcl`. I don't think we need to worry about updates to other resources in this tutorial, but I'm open to suggestions.